src/BloomExe/web/BloomServer.cs

```diff
@@ -1448,6 +1456,33 @@ private void EnqueueIncomingRequests()
         {
             while (_listener.IsListening)
             {
+                // We've found that sometimes one of our worker threads just dies. One way to force it to happen is to
+                // uncomment the block of code that converts requests for .map files into 404s.
+                // We know of no reason for a thread to die except for throwing an uncaught exception, and the method
+                // in which this thread loops catches all exceptions that it can, and the handler does not fire in
+                // this situation. Conceivably it is something like a stack overflow exception that can't be caught.
+                // It's very bad if all our server threads die; Bloom freezes up and can't even quit (in edit mode) because
+                // that requires a server request to obtain the page content. So we detect dead threads here and replace them.
+                // This is not very satisfactory. We don't know what task if any was left incomplete by the dead thread,
+                // nor whether it might have incremented but not decremented one of our counts of threads-in-use.
+                // But it's better than freezing up.
+                var deadThreads = _workers.Where(kvp => !kvp.Value.IsAlive);
+                foreach (var kvp in deadThreads)
+                {
+                    //thread.Join(); Copilot suggested this but I don't think you can join a dead thread???
+                    // Do we want a more drastic report? We don't know what went wrong with the thread, so a report
+                    // is unlikely to be very informative. But it's remotely possible that it damaged some data.
+                    // Could this be related to the wipeout bug?
+                    Debug.WriteLine(
```

Comment:
Should probably be a Logger.Event() so that we can see if this is happening a lot to someone.